homeassistant/components/airvisual/sensor.py

```diff
@@ -59,60 +59,82 @@
 SENSOR_KIND_TEMPERATURE = "temperature"
 SENSOR_KIND_VOC = "voc"
 
-GEOGRAPHY_SENSORS = [
-    (SENSOR_KIND_LEVEL, "Air Pollution Level", "mdi:gauge", None),
-    (SENSOR_KIND_AQI, "Air Quality Index", "mdi:chart-line", "AQI"),
-    (SENSOR_KIND_POLLUTANT, "Main Pollutant", "mdi:chemical-weapon", None),
-]
+GEOGRAPHY_SENSOR_DESCRIPTIONS = (
+    SensorEntityDescription(
+        key=SENSOR_KIND_LEVEL,
+        name="Air Pollution Level",
+        icon="mdi:gauge",
+    ),
+    SensorEntityDescription(
+        key=SENSOR_KIND_AQI,
+        name="Air Quality Index",
+        icon="mdi:chart-line",
+        native_unit_of_measurement="AQI",
+    ),
+    SensorEntityDescription(
+        key=SENSOR_KIND_POLLUTANT,
+        name="Main Pollutant",
+        icon="mdi:chemical-weapon",
+    ),
+)
 GEOGRAPHY_SENSOR_LOCALES = {"cn": "Chinese", "us": "U.S."}
 
-NODE_PRO_SENSORS = [
-    (SENSOR_KIND_AQI, "Air Quality Index", None, "mdi:chart-line", "AQI"),
-    (SENSOR_KIND_BATTERY_LEVEL, "Battery", DEVICE_CLASS_BATTERY, None, PERCENTAGE),
-    (
-        SENSOR_KIND_CO2,
-        "C02",
-        DEVICE_CLASS_CO2,
-        None,
-        CONCENTRATION_PARTS_PER_MILLION,
+NODE_PRO_SENSOR_DESCRIPTIONS = (
```

Comment:
```suggestion
NODE_PRO_SENSOR_DESCRIPTIONS: tuple[SensorEntityDescription, ...] = (
```

Reply:
Why? `mypy` doesn't complain without this, which it can infer the type.

Reply:
It's a nice to have, but yes not strictly necessary.